common/djangoapps/util/enterprise_helpers.py

```diff
@@ -120,3 +124,62 @@ def insert_enterprise_pipeline_elements(pipeline):
 
     for index, element in enumerate(additional_elements):
         pipeline.insert(insert_point + index, element)
+
+
+def get_enterprise_customer_logo_url(request):
+    """
+    Client API operation adapter/wrapper
+    """
+
+    if not enterprise_enabled():
+        return None
+
+    parameter = get_enterprise_branding_filter_param(request)
+    if not parameter:
+        return None
+
+    provider_id = parameter.get('provider_id', None)
+    ec_uuid = parameter.get('ec_uuid', None)
+
+    if provider_id:
+        branding_info = enterprise_api.get_enterprise_branding_info_by_provider_id(provider_id=provider_id)
+    elif ec_uuid:
+        branding_info = enterprise_api.get_enterprise_branding_info_by_ec_uuid(ec_uuid=ec_uuid)
+
+    logo_url = None
+    if branding_info and branding_info.logo:
+        logo_url = branding_info.logo.url
+
+    return logo_url
+
+
+def set_enterprise_branding_filter_param(request, provider_id=None):
+    """
+    Setting 'EC_BRANDING_FILTER_PARAM' in session. 'EC_BRANDING_FILTER_PARAM' either be provider_id or ec_uuid.
+    e.g. {provider_id: 'xyz'} or {ec_src: enterprise_customer_uuid}
+    """
+    ec_uuid = request.GET.get('ec_src', None)
+    if EC_BRANDING_FILTER_PARAM in request.session and (provider_id or ec_uuid):
```

Comment:
Well, at least the way this is currently architectured, setting a new value will remove any existing value anyway - we're setting the value of the session variable, not updating it. I'd prefer to remove this if we're not using it, especially because as part of a larger block, it can't be effectively tested to ensure it's functioning correctly.